app/components/contributors/edit_component.html.erb

```diff
@@ -28,19 +28,25 @@
   <div data-contributors-target="personName">
     <%= render Elements::Forms::LabelComponent.new(form:, field_name: :with_orcid, label_text: t('works.edit.fields.contributors.orcid_or_name.label')) %><br>
     <div class="form-check">
-      <%= form.radio_button :with_orcid, true, class: 'form-check-input' %>
+      <%= form.radio_button :with_orcid, true, class: 'form-check-input', checked: true %>
       <%= form.label :with_orcid_true, t('works.edit.fields.contributors.orcid.label'), class: 'form-check-label' %>
-      <%= tag.div class: orcid_container_classes do %>
-        <%= render Elements::ButtonComponent.new(label: t('works.edit.fields.contributors.use_orcid_button.label'), classes: 'btn btn-outline-primary', data: { action: 'contributors#useOrcid' }) if orcid.present? %>
+      <div class="mb-4 field-container input-group">
         <%= render Elements::Forms::LabelComponent.new(form:, field_name: :orcid, hidden_label: true) %>
-        <%= form.text_field :orcid, class: 'form-control', data: { contributors_target: 'orcidField' } %>
-      <% end %>
+        <span class="input-group-text">
+          <%= Settings.orcid.url %>
+        </span>
```

Comment:
Use a bootstrap input group to display the ORCID prefix on the text field. Use the URI-less form of the ORCID throughout the app.